background.js

```diff
@@ -3,7 +3,8 @@ const code = `(() => {
   for (let i = 0; i < elements.length; i++) {
     const el = elements[i]
     const pos = window.getComputedStyle(el).position
-    if (pos === 'fixed' || pos === 'sticky') {
+	const style = window.getComputedStyle(el);
+    if ((pos === 'fixed' || pos === 'sticky') && style.display !== 'none') {
```

Comment:
Please fix the indentation style to match, and refactor to only call `window.getComputedStyle(el)` once.